content/integrate/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Integrations and frameworks
+title: Libraries and tools
 description: 
 linkTitle: Integrate
----
\ No newline at end of file
+---
```

Comment:
That's the correct file. I'm happy to make the change to something like:

Search libraries and tools                                 Filter by type

or whatever.

Reply:
Thanks for this comment, much appreciated! I'm going to leave this to you David because the text isn't plain text in the file and I'm not sure how it should be changed actually.